e2e-tests/cypress/e2e/journeys/insurance/check-your-answers/policy-and-exports/change-your-answers/change-your-answers-policy-and-exports-multiple.spec.js

```diff
@@ -0,0 +1,325 @@
+      CONTRACT_POLICY: {
+        REQUESTED_START_DATE,
+        CREDIT_PERIOD_WITH_BUYER,
+        POLICY_CURRENCY_CODE,
+        MULTIPLE: { TOTAL_MONTHS_OF_COVER, TOTAL_SALES_TO_BUYER, MAXIMUM_BUYER_WILL_OWE },
+      },
+    },
+  },
+} = FIELD_IDS;
+
+const { taskList, policyCurrencyCodeFormField } = partials.insurancePartials;
+
+const task = taskList.submitApplication.tasks.checkAnswersAndSubmit;
+
+const { summaryList } = checkYourAnswersPolicyAndExports;
+
+context('Insurance - Change your answers - Policy and exports - multiple contract policy - Summary List', () => {
+  let url;
+  let referenceNumber;
+
+  const getFieldVariables = (fieldId) => ({
+    route: MULTIPLE_CONTRACT_POLICY_CHECK_AND_CHANGE,
+    checkYourAnswersRoute: TYPE_OF_POLICY,
+    newValueInput: '',
+    fieldId,
+    referenceNumber,
+    summaryList,
+    changeLink: summaryList.field(fieldId).changeLink,
+  });
+
+  before(() => {
+    cy.completeSignInAndGoToApplication().then((refNumber) => {
+      referenceNumber = refNumber;
+      cy.completePrepareApplicationMultiplePolicyType();
+
+      task.link().click();
+
+      // to get past eligibility check your answers page
+      submitButton().click();
+
+      url = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${TYPE_OF_POLICY}`;
+
+      cy.url().should('eq', url);
+    });
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+  });
+
+  after(() => {
+    cy.deleteAccount();
+  });
+
+  describe('multiple policy type answers', () => {
+    describe(REQUESTED_START_DATE, () => {
+      const fieldId = REQUESTED_START_DATE;
+      let fieldVariables = getFieldVariables(fieldId);
+
+      const newAnswer = {
+        ...application.POLICY_AND_EXPORTS[fieldId],
+        year: application.POLICY_AND_EXPORTS[fieldId].year + 1,
+      };
+
+      describe('when clicking the `change` link', () => {
+        beforeEach(() => {
+          cy.navigateToUrl(url);
+        });
+
+        it(`should redirect to ${MULTIPLE_CONTRACT_POLICY_CHECK_AND_CHANGE}`, () => {
+          cy.navigateToUrl(url);
+          fieldVariables = getFieldVariables(fieldId);
+
+          checkChangeLinkUrl(fieldVariables, referenceNumber);
+        });
+      });
+
+      describe('form submission with a new answer', () => {
+        beforeEach(() => {
+          cy.navigateToUrl(url);
+
+          summaryList.field(fieldId).changeLink().click();
+
+          fieldVariables.newValueInput = newAnswer.year;
+
+          changeAnswerField(fieldVariables, multipleContractPolicyPage[fieldId].yearInput());
+        });
+
+        it(`should redirect to ${TYPE_OF_POLICY}`, () => {
+          cy.assertChangeAnswersPageUrl(referenceNumber, TYPE_OF_POLICY, fieldId);
+        });
+
+        it('should render the new answer', () => {
+          fieldVariables.newValue = formatDate(createTimestampFromNumbers(newAnswer.day, newAnswer.month, newAnswer.year));
+          checkChangeAnswerRendered(fieldVariables);
```

Comment:
Could we move `checkChangeAnswerRendered`, `changeAnswerField` and `checkChangeLinkUrl` into cypress commands? Or future PR?

Reply:
future PR please as I can do more than 1 file at the same time

Reply:
https://ukef-dtfs.atlassian.net/browse/EMS-1296